agent/http.go

```diff
@@ -135,11 +148,39 @@ func (s *HTTPServer) handler(enableDebug bool) http.Handler {
 		handleFuncMetrics("/debug/pprof/symbol", pprof.Symbol)
 	}
 
-	// Use the custom UI dir if provided.
-	if s.agent.config.UIDir != "" {
-		mux.Handle("/ui/", http.StripPrefix("/ui/", http.FileServer(http.Dir(s.agent.config.UIDir))))
-	} else if s.agent.config.EnableUI {
-		mux.Handle("/ui/", http.StripPrefix("/ui/", http.FileServer(assetFS())))
+	if s.IsUIEnabled() {
+		new_ui := false
+		switch strings.ToLower(os.Getenv("CONSUL_UI_BETA")) {
+			case "1":
+				fallthrough
+			case "true":
+				fallthrough
+			case "yes":
+				new_ui = true
+			default:
+				new_ui = false		
+		}
```

Comment:
Can just use:
```go
new_ui, err := strconv.ParseBool(os.Getenv("CONSUL_UI_BETA"))
if err != nil {
  // Invalid/empty value is treated as false
  new_ui = false
}
```

https://golang.org/pkg/strconv/#ParseBool

Reply:
Yeah, I can do that for simplicity. IMO ParseBool should also handle y,Y,Yes etc as true values but it doesn't. Many env variables I have used in other products and had in my own in the past like yes/no values for things. I was trying to keep it intuitive. Then again people using Consul are probably used to how we handle things anyways.